packages/actor-rdf-parse-rdfxml/lib/ActorRdfParseRdfXml.ts

```diff
@@ -0,0 +1,23 @@
+import {ActorRdfParseFixedMediaTypes, IActionRdfParse, IActorRdfParseFixedMediaTypesArgs,
+  IActorRdfParseOutput} from "@comunica/bus-rdf-parse";
+import {ActionContext} from "@comunica/core";
+import {RdfXmlParser} from "rdfxml-streaming-parser";
+
+/**
+ * A comunica RDF/XML RDF Parse Actor.
+ */
+export class ActorRdfParseRdfXml extends ActorRdfParseFixedMediaTypes {
+
+  constructor(args: IActorRdfParseFixedMediaTypesArgs) {
+    super(args);
+  }
+
+  public async runHandle(action: IActionRdfParse, mediaType: string, context: ActionContext)
+    : Promise<IActorRdfParseOutput> {
+    action.input.on('error', (e) => quads.emit('error', e));
+    const quads = action.input.pipe(new RdfXmlParser());
+    return {
+      quads,
+      triples: mediaType === 'application/rdf+xml'};
```

Comment:
`triples` is always true for RDF/XML, so just set `triples: true`, and you can ignore the `mediaType`.
Also, put the `}` on a new line.